src/test/scripts/README.md

```diff
@@ -0,0 +1,25 @@
+
+# check-merge-to-enterprise Job
+
+When you open a PR on community, if it creates a conflict with enterprise-master, the check-merge-to-enterprise will fail. Say your branch name is `X`, we will refer to `X` on community as `community/X` and on enterprise as `enterprise/X`. In order to make the job pass:
+
+- You first need to get approval from your reviewer for `community/X`. Only follow the next steps after you are about to merge the branch to community master. Otherwise, the `check-merge-to-enterprise` check might falsely pass. (For example when you added new commits to `community/X` but forgot to update `enterprise/X`).
+- You need to synchronize the same branch, `X` locally on enterprise (ideally don't push yet), and merge `enterprise-master` to `enterprise/X`. Solve any conflicts( and make sure to remove any parts that should not be in enterprise even though it doesn't have a conflict), on enterprise repository:
+
+```bash
+git checkout enterprise-master
+git pull # make sure enterprise-master in your local is up-to-date
+git checkout X
+git merge enterprise-master
+```
+
+- You should push this to enterprise and open a PR so that the job on community will see this branch.
```

Comment:
```suggestion
- You should push this to enterprise and open a PR so that the job on community will see this branch. Also this will trigger CI to verify changes are correct.
```